views/fe-utils/siteTooltipContent.js

```diff
@@ -0,0 +1,30 @@
+import { TOTAL_LABEL, NA } from '../../constants/vars'
+import { formatAsPercentage } from './formatAsPercentageUtil'
+
+export const siteTooltipContent = (siteData) => {
+  const tooltipData = []
+  let siteTotalCount = 0
+  let siteTotalPercent = 0
+
+  siteData
+    .filter(({ name }) => name !== NA)
+    .forEach(({ name: valueName, payload }) => {
+      if (valueName !== NA) {
+        const valuePercent = payload.percentages[valueName]
+        const valueColumn = `${payload.counts[valueName]}/(${formatAsPercentage(
+          valuePercent
+        )})`
```

Comment:
this should be `COUNT / TARGET (PERCENT%)`. Right now it looks like `COUNT/(PERCENT%)`

Reply:
now it looks like `COUNT/TARGET(PERCENT%)` not `COUNT / TARGET (PERCENT%)`